src/client.ts

```diff
@@ -3424,6 +3424,23 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
         return this.getIgnoredUsers().includes(userId);
     }
 
+    /**
+     * Gets the list of currently ignored invites.
+     */
+    public getIgnoredInvites(): IgnoredInvites {
+        const event = this.getAccountData(EventType.IgnoredInvites);
```

Comment:
I haven't been able to find exactly where this is implemented. However, as far as I understand, `this.getAccountData` should be fast (because everything is cached to memory).

Also note that you can subscribe to `client.on(ClientEvent.AccountData, event => { ... })` to be informed whenever a piece of account data changes.